src/devices/zemismart.ts

```diff
@@ -105,6 +105,40 @@ const definitions: Definition[] = [
             await reporting.onOff(device.getEndpoint(2));
         },
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_7eue9vhc', '_TZE200_bv1jcqqu']),
+        model: 'ZM25RX-08/30',
+        vendor: 'Zemismart',
+        description: 'Tubular motor',
+        onEvent: tuya.onEvent(),
+        configure: tuya.configureMagicPacket,
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        options: [exposes.options.invert_cover()],
+        exposes: [
+            e.text('work_state', ea.STATE),
+            e.cover_position().setAccess('position', ea.STATE_SET),
+            e.battery(),
+            e.enum('program', ea.SET, ['SET BOTTOM', 'SET UPPER', 'RESET']).withDescription('Set the upper/bottom limit'),
+            e.enum('program', ea.SET, ['UPPER', 'UPPER MICRO', 'LOWER', 'LOWER MICRO']).withDescription('Steps control (ignores set limit)'),
```

Comment:
Although I understand why you splitted them, this will not work with Home Assistant integration since it is the same expose, please merge them.

Reply:
I split them into program and click_control, but have to add [fallbackValue](https://github.com/Koenkk/zigbee-herdsman-converters/pull/6341/files#diff-d45f20d30a43dc3e172b16bd55872492f4867c8831c5e03a8be9be65f1cd72d5R403) into lookup converter. Another way just keep all values in both datapoints
```
        [101, 'program', tuya.valueConverterBasic.lookup({
          'set_bottom': tuya.enum(0), 'set_upper': tuya.enum(1), 'reset': tuya.enum(4),
          'lower': tuya.enum(2), 'upper': tuya.enum(3),
          'lower_micro': tuya.enum(5), 'upper_micro': tuya.enum(6),
        })],
        [101, 'click_control', tuya.valueConverterBasic.lookup({
          'set_bottom': tuya.enum(0), 'set_upper': tuya.enum(1), 'reset': tuya.enum(4),
          'lower': tuya.enum(2), 'upper': tuya.enum(3),
          'lower_micro': tuya.enum(5), 'upper_micro': tuya.enum(6),
        })],
```